experiments/sim_result_visualization.Rmd

```diff
@@ -0,0 +1,218 @@
+---
+title: "Visualization of simulation results"
+date: "3/28/2022"
+output: html_document
```

Comment:
is this supposed to be vignette that users can read online? if so it should go in vignettes/. for that extra packages needs to be installed on GH so ideally you should be able to plot everything below using just ggplot2 without loading 4 extra libraries, we can talk more in person today

Reply:
it's just work in progress code generating the figures for the paper (saved in .Rmd rather than .R for convenience while editing), it won't be part of the user vignette